src/entity/Pickup.ts

```diff
@@ -58,6 +58,8 @@ export type IPickup = HasSpace & {
   // Pickups optionally have a "time limit" and will disappear after this many turns
   turnsLeftToGrab?: number;
   text?: PIXI.Text;
+  // Boosts the effect of the pickup, default 1
+  power: number;
```

Comment:
The copy for each pickup now needs to be changed so that it accounts for the power level.  For example merged health potions still say "heals 50" when it heals 100.

This will mean pickup description will have to be changed to a function that returns a `Localizable` array instead of just the array and the invocation in PlanningView line 907 will have to change from `i18n(globalThis.selectedPickup.description)` to `i18n(globalThis.selectedPickup.description(globalThis.selectedPickup)`.
That way the pickup instance will be available in the description can you can change the descriptions to, for example, `     description: pickup => ['health potion description', (pickup.power * healthPotionRestoreAmount).toString()],`